package.json

```diff
@@ -15,7 +15,7 @@
     "d3": ">= 3.0.0"
   },
   "scripts": {
-    "es5": "rollup --config",
+    "es5": "rollup --config && sed -i 's/d3$1/d3/' build/webcharts.js",
```

Comment:
Not sure this is needed ... Is the fact that `rollup` creates an alias (`d3$1`) causing any problems?